pages/api/utils/api-utils.js

```diff
@@ -16,28 +16,53 @@
 
 import * as nodeFetch from "node-fetch";
 
-export const getRodeUrl = () =>
-  process.env.RODE_URL || "http://localhost:50051";
-
-const fetch = (endpoint, method, body) => {
+const fetch = ({ endpoint, method, body, accessToken }) => {
```

Comment:
Picky: would it be a good idea to keep the `getRodeUrl()` function and have that call out fo `config.get("rode.url")` so if the config key ever changes, the change is only needed in one place?

The `get`/`post`/etc utils could also call to get this value since we're not calling anything other than Rode right now but that could change in the future, I suppose.

Reply:
I don't feel too strongly, but I think I'd rather have a wrapper around the `get/`patch`/`del`/`post` utils that is pre-configured to call Rode. That would eliminate a lot of these call sites for the config and maybe centralize some of the error handling.